IBAnimatable/ContainerTransition.swift

```diff
@@ -49,13 +49,13 @@ public class ContainerTransition: NSObject {
       return
     }
     
-    viewControllers = [UITransitionContextFromViewControllerKey: unwrappedFromViewController,
-                       UITransitionContextToViewControllerKey: toViewController]
-    views = [UITransitionContextFromViewKey: unwrappedFromViewController.view,
-             UITransitionContextToViewKey: toViewController.view]
+    viewControllers = [UITransitionContextViewControllerKey.from.rawValue: unwrappedFromViewController,
```

Comment:
If we let `UITransitionContextViewControllerKey` comform to `Equatable` and implement `func ==(lhs: Self, rhs: Self) -> Bool`, can we use it as the key for the dictionary. Then we don't need to use `.rawValue`?